lib/clients/rtm/client.js

```diff
@@ -433,21 +414,21 @@ RTMClient.prototype.handleWsMessage = function handleWsMessage(wsMsg) {
   try {
     message = JSON.parse(wsMsg);
   } catch (err) {
-    // TODO(leah): Emit an event here?
-    this.logger('debug', 'unable to parse message: ' + err);
+    this.logger('error', 'Unable to parse message: ' + err);
     return;
   }
 
   if (includes(RTM_CLIENT_INTERNAL_EVENT_TYPES, message.type)) {
     this._handleWsMessageInternal(message.type, message);
   } else {
+    this._maybeKeepAlive(message);
```

Comment:
so the idea is, pong messages will still keep it alive, but if we _don't_ receive a pong message because we are being flooded with so many other legitimate messages, we'll treat those messages like a pong and not close the socket